docs/Concepts/Pruning.md

```diff
@@ -0,0 +1,13 @@
+description: Pruning
+<!--- END of page meta data -->
+
+# Pruning
+
+Use pruning to reduce storage required for the chain. As the number of transactions increases, the world state grows. Pruning removes state tries that aren't required.
```

Comment:
`chain` -> `world-state size`

The world state isn't guaranteed to grow along with transactions so I would just remove that sentence.

`state tries` -> `state-trie nodes`

Reply:
Updated.